src/System.Management.Automation/engine/hostifaces/LocalConnection.cs

```diff
@@ -897,6 +889,18 @@ private void DoCloseHelper()
             //Raise Event
             RaiseRunspaceStateEvents();
 
+            // Close all open runspaces to ensure that any associated pipeline thread is released.
+            if (closeAllOpenRunspaces)
```

Comment:
I'm thinking about ISE here. Each tab is an interactive host with a different Runspace, so which one should be the default Runspace? 
A user can type `ctrl+t` and `ctrl+w` to create a new tab and close an existing tab, so it's very common that the first tab gets closed by `ctrl+w` later. If the first tab's Runspace is set to be the default, then closing that tab would terminate all tabs ...